src/main/java/propertyFiles/codeMeta20.properties

```diff
@@ -1,12 +1,12 @@
 metadatablock.name=codeMeta20
-metadatablock.displayName=Software Metadata (CodeMeta v2.0)
+metadatablock.displayName=Software Metadata (CodeMeta v3.0)
 metadatablock.displayFacet=Software
 datasetfieldtype.codeVersion.title=Software Version
 datasetfieldtype.codeVersion.description=Version of the software instance, usually following some convention like SemVer etc.
 datasetfieldtype.codeVersion.watermark=e.g. 0.2.1 or 1.3 or 2021.1 etc
 datasetfieldtype.developmentStatus.title=Development Status
 datasetfieldtype.developmentStatus.description=Description of development status, e.g. work in progress (wip), active, etc. See repostatus.org for more information.
-datasetfieldtype.developmentStatus.watermark=                                                                                                                       Development Status
+datasetfieldtype.developmentStatus.watermark=
```

Comment:
What's going on here? 🤔

Reply:
a watermark for a controlled vocab field does not really make sense, right?

Reply:
I suppose not! Thanks! Resolving.